weave/runtime.nim

```diff
@@ -43,6 +43,7 @@ proc init*(_: type Weave) =
     workforce() = int32 countProcessors()
 
   ## Allocation of the global context.
+  globalCtx.auxiliaryInit = unsafeAddr auxiliary
```

Comment:
auxiliary is already an address

Reply:
I did the pointer indirection because the compiler was complaining about GC/thread safety in the global context type, didn't realize it was because of the closure possibility